src/_includes/common-form-fields.html

```diff
@@ -0,0 +1,134 @@
+
+   ========================================================================== #}
+
+{% macro state(size='1-3', required=false, disabled=false) %}
+  {% from 'macros/form.html' import select as select %}
+  {{ select('State',
+            'Select state',
+            [ 'Alabama',
+              'Alaska',
+              'Arizona',
+              'Arkansas',
+              'California',
+              'Colorado',
+              'Connecticut',
+              'Delaware',
+              'District of Columbia',
+              'Florida',
+              'Georgia',
+              'Hawaii',
+              'Idaho',
+              'Illinois',
+              'Indiana',
+              'Iowa',
+              'Kansas',
+              'Kentucky',
+              'Louisiana',
+              'Maine',
+              'Montana',
+              'Nebraska',
+              'Nevada',
+              'New Hampshire',
+              'New Jersey',
+              'New Mexico',
+              'New York',
+              'North Carolina',
+              'North Dakota',
+              'Ohio',
+              'Oklahoma',
+              'Oregon',
+              'Maryland',
+              'Massachusetts',
+              'Michigan',
+              'Minnesota',
+              'Mississippi',
+              'Missouri',
+              'Pennsylvania',
+              'Rhode Island',
+              'South Carolina',
+              'South Dakota',
+              'Tennessee',
+              'Texas',
+              'Utah',
+              'Vermont',
+              'Virginia',
+              'Washington',
+              'West Virginia',
+              'Wisconsin',
+              'Wyoming'
+            ],
+            size,
+            required,
+            disabled) }}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   common_form_fields.date_picker()
+
+   ==========================================================================
+
+   Description:
+
+   Returns a date picker
+
+   legend: Name of the field.
+
+   required: Whether the field is required. Defaults to false.
+
+   disabled: Whether the field is disabled. Defaults to false.
+
+   ========================================================================== #}
+
+{% macro date_picker(legend='Select Date Range', required=false, disabled=false) %}
+   <fieldset class="u-reset">
+                <div class="form-l_col form-l_col-1 u-mb0">
+                    <legend class="form-label-header">
+                        {{ legend }}
+                    </legend>
+                </div>
+                <div class="form-l_col form-l_col-1-2">
+                    <label class="micro-copy micro-copy__large"
+                           for="foia-request_starting-date">
+                        From:
+                    </label>
+                    <input class="input__long"
+                           id="foia-request_starting-date"
```

Comment:
Good catch! I'll change it to something generic unless you think we'll have a situation when there's two on a page in which case I can slugify the legend.